packages/xr/src/feature/trackable/XRTrackableFeature.ts

```diff
@@ -22,6 +27,28 @@ export abstract class XRTrackableFeature<T extends XRTracked, K extends XRReques
   protected _statusSnapshot: Record<number, XRTrackingState> = {};
   private _listeners: ((added: readonly T[], updated: readonly T[], removed: readonly T[]) => void)[] = [];
 
+  /**
+   * Returns the prefab that is automatically mounted when the object is tracked.
+   *
+   * @remarks
+   * If you set a Prefab, a Prefab instance will be created when the object is tracked,
+   * otherwise, an Entity will be used instead
+   */
+  get prefab(): PrefabResource {
```

Comment:
Specialized resource types should not be used as interfaces

Reply:
Done.